taxi/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from . import views
```

Comment:
split different groups of imports + it's better to use absolute import path
The reason is that when you have import like
`from taxi import views`
and you move this file somewhere, Pycharm will recommend to update imports automatically
However if you use relative paths, it won't work in this way